data/config/ndh.json

```diff
@@ -123,7 +123,7 @@
     },
     "login": {
       "title": "NIAID Data Hub",
-      "subTitle": "Explore, Analyze, and Share Data",
+      "subTitle": "search, compare, and download data",
```

Comment:
can you change this back to "Explore, Analyze, and Share Data" ?

Reply:
Prod use this so I copied them

Reply:
:O I'll change prod - they should all be "Explore, Analyze, and Share Data" now